raiden/network/transport/matrix/transport.py

```diff
@@ -1520,7 +1520,7 @@ def retry_api_call(
                 last_ex = e
             finally:
                 if self._stop_event.wait(retry_interval):
-                    return return_value
+                    return return_value  # noqa: B012
```

Comment:
@ulope I didn't want to change a lot of code here, so I just allowed it. The original message is:
```
B012 return/continue/break inside finally blocks cause exceptions to be silenced. Exceptions should be silenced in except blocks. Control statements can be moved outside the finally block.
```